src/test/java/org/sagebionetworks/web/unitclient/widget/entity/menu/v2/ActionMenuWidgetImplTest.java

```diff
@@ -92,8 +96,16 @@ public void testReset() {
 		widget.setActionListener(Action.CHANGE_ENTITY_NAME, mockActionListener);
 		// Now reset the the widget
 		widget.reset();
-		// nothing should happen here.
-		widget.onAction(Action.CHANGE_ENTITY_NAME);

```

Comment:
We may start seeing new exceptions since invoking actions with no listeners was previously permitted.